
From: Stephen Tweedie <sct@redhat.com>

All ext2/3 ioctls apart from these ones use 'f' as the root char for their
macro-generated ioctl numbers; make reservations consistent with this.

Signed-off-by: Stephen Tweedie <sct@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/ext3_fs.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN include/linux/ext3_fs.h~ext3-reservations-renumber-the-ext3-reservations-ioctls include/linux/ext3_fs.h
--- 25/include/linux/ext3_fs.h~ext3-reservations-renumber-the-ext3-reservations-ioctls	Tue Sep  7 15:36:12 2004
+++ 25-akpm/include/linux/ext3_fs.h	Tue Sep  7 15:36:13 2004
@@ -207,8 +207,8 @@ struct ext3_group_desc
 #ifdef CONFIG_JBD_DEBUG
 #define EXT3_IOC_WAIT_FOR_READONLY	_IOR('f', 99, long)
 #endif
-#define EXT3_IOC_GETRSVSZ		_IOR('r', 1, long)
-#define EXT3_IOC_SETRSVSZ		_IOW('r', 2, long)
+#define EXT3_IOC_GETRSVSZ		_IOR('f', 5, long)
+#define EXT3_IOC_SETRSVSZ		_IOW('f', 6, long)
 
 /*
  * Structure of an inode on the disk
_
